pom.xml

```diff
@@ -121,9 +122,9 @@
   <dependencies>
         <!-- Spigot -->
         <dependency>
-          <groupId>com.destroystokyo.paper</groupId>
+          <groupId>io.papermc.paper</groupId>
           <artifactId>paper-api</artifactId>
-          <version>${spigot.version}</version>
+          <version>1.17.1-R0.1-SNAPSHOT</version>
```

Comment:
I think this was this changed accidentally?

Reply:
Correct. Should remain unchanged. Had initial problems due to PICNIC error.